examples/benchmarking/M5-forecasting.ipynb

```diff
@@ -0,0 +1,3944 @@
+{
```

Comment:
<p>For all of the following code cells please always add a markdown cell explaining what you are doing. This helps the reader to understand better what you are doing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/7113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>